pkg/apis/elasticsearch/v1/fields.go

```diff
@@ -18,8 +18,10 @@ const (
 
 	NodeName = "node.name"
 
-	PathData = "path.data"
-	PathLogs = "path.logs"
+	PathData                 = "path.data"
+	PathLogs                 = "path.logs"
+	ShardAwarenessAttributes = "cluster.routing.allocation.awareness.attributes"
```

Comment:
cosmetic nit: maybe add a new line between those new fields and the existing since they address different purposes in the configuration ?